src/app/component/Field/Field.container.js

```diff
@@ -10,16 +10,16 @@
  */
 
 import { connect } from 'react-redux';
-import { FieldDispatcher } from 'Store/Field';
+import { CountryDispatcher } from 'Store/Country';
 import Field from './Field.component';
 
 const mapStateToProps = state => ({
-    countries: state.FieldReducer.countries
+    countries: state.CountryReducer.countries
 });
 
 const mapDispatchToProps = dispatch => ({
     getCountriesList() {
-        return FieldDispatcher.getCountriesList(dispatch);
+        return CountryDispatcher.getCountriesList(dispatch);
```

Comment:
Why is this one here? I do not want to request any countries from Field! Field is for display. Please do not connect or request, or do anything else in it except displaying specific input type! No. Please remove the logic of `getCountriesList` from Field at all. We discussed that multiple times!

Reply:
That was already fixed in Checkout-Country-Select.